static/src/javascripts/projects/commercial/modules/consentless/define-slot.ts

```diff
@@ -1,16 +1,29 @@
 import { renderAdvertLabel } from '../dfp/render-advert-label';
 
-const getOptOutSlotName = (dfpSlotName: string): string => {
-	if (dfpSlotName.includes('top-above-nav')) {
-		return 'homepage-lead';
+const isArticle = window.guardian.config.page.contentType === 'Article';
+const isFront = window.guardian.config.page.isFront;
+
+const getOptOutSlotName = (slotName: string, inlineId?: number): string => {
+	// front, liveblog, article inlines all have slightly different mappings, so we prefix them
+	if (slotName === 'inline') {
+		if (isArticle) {
+			// article inlines are suffixed with 1 for inline1 and 2-plus for all other inlines as they have different size mappings
+			return `article-inline${inlineId === 1 ? '1' : '2-plus'}`;
```

Comment:
I don't know if we have much choice but adding the content type to the slot names feels less than ideal.

So in opt-out world I assume there is no method to switch the size mappings used by a line item via targeting params e.g. `contentType: 'front'` ?

Reply:
The only thing I can see is we can set some targeting on the slot definitions, which I assume would be used to match the slot via `ootag.addParameterForSlot()`, I don't think that helps us though?🤔

For now we could have just 'inline' only MPU size for article inline1 and all inlines on other pages, and inline2plus on articles which also has the skyscraper etc. But who knows what sizes may be added in the future.